packages/ui/package.json

```diff
@@ -7,7 +7,8 @@
   "exports": {
     ".": "./index.tsx",
     "./components/toast": "./components/toast/index.tsx",
-    "./components/icon": "./components/icon/index.ts"
+    "./components/icon": "./components/icon/index.ts",
+    "./components/icon/Discord": "./components/icon/Discord.tsx"
```

Comment:
you could have just exported Discord.tsx from icon/inedex.js

Reply:
I tried but for some reason it didn't work. The issue was it is trying to export from lucide react